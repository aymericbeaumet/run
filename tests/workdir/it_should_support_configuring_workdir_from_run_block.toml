[[run]]
workdir = "./fixtures/b"
name = "ls"
cmd = ["coreutils", "ls"]
