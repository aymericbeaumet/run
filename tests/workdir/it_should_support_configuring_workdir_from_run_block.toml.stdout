[ls] b1
[ls] b2
[ls] b3